torch/onnx/symbolic_opset17.py

```diff
@@ -15,5 +15,19 @@
     SequenceMap
 """
 
+from torch.onnx import symbolic_helper
+
 # EDITING THIS FILE? READ THIS FIRST!
 # see Note [Edit Symbolic Files] in symbolic_helper.py
+
+
+@symbolic_helper.parse_args("v", "is", "v", "v", "f", "i")
+def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
+    return g.op(
+        "LayerNormalization",
+        input,
+        weight,
+        bias,
+        epsilon_f=eps,
+        axis_i=len(normalized_shape),
```

Comment:
Can you write a brief comment inline explaining what axis is and why len(normalized_shape) is the same thing?

Reply:
Good question, I think particularly in this test case, the value happens to match...

Reply:
I was referencing on this:
https://github.com/pytorch/pytorch/blob/14093b5979cb5c0b777e3920819ab8252eb6d3ea/torch/onnx/symbolic_opset9.py#L2309

And according to https://github.com/onnx/onnx/blob/main/docs/Changelog.md#LayerNormalization-17
>axis : int (default is -1)
The first normalization dimension. If rank(X) is r, axis' allowed range is [-r, r]. Negative value means counting dimensions from the back.

I tried different params for layer_norm and input, but it works so far. @BowenBao is there a test case you have on mind that might fail?

Reply:
What are the combinations you tried?
I suspect it should be `axis_i = -len(normalized_shape)`. However for this current test case, both `axis_i = 2` and `axis_i = -2` maps to the same dimension. So I'd recommend trying alternate the rank
```python
     model = torch.nn.LayerNorm([10, 10, 10]) 
     x = torch.randn(20, 5, 10, 10, 10) 
     self.run_test(model, x) 
```

Reply:
I see

Reply:
Done